frontend/app/views/spree/checkout/_terms_and_conditions.en.html.erb

```diff
@@ -0,0 +1,64 @@
+____________________________________________________________
```

Comment:
> In this way we prevent inserting the html code (link, images ecc...)

I don't see why this is a concern. Nobody is forcing a developer to add those to this file.

> The terms and conditions should be in a format that can be printed or saved

There is no reason a textarea is better for this. Using a div with `overflow: scroll` we even have the opportunity to expand it in a print-only stylesheet if that is a real concern. The "text only" feel of a textarea could be duplicated using `white-space: pre` for sites which really want that, and sites which already have an HTML version of their terms and conditions can use that.